packages/ui/components/badge/UpgradeTeamsBadge.tsx

```diff
@@ -1,16 +1,15 @@
 import Link from "next/link";
 
-import { useHasPaidPlan } from "@calcom/lib/hooks/useHasPaidPlan";
+import { useHasTeamPlan } from "@calcom/lib/hooks/useHasPaidPlan";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 
 import { Tooltip } from "../tooltip";
 import { Badge } from "./Badge";
 
 export const UpgradeTeamsBadge = function UpgradeTeamsBadge() {
   const { t } = useLocale();
-  const { hasPaidPlan } = useHasPaidPlan();
-
-  if (hasPaidPlan) return null;
+  const { hasTeamPlan } = useHasTeamPlan();
+  if (hasTeamPlan) return null;
```

Comment:
Can you explain why you changed the hook here

Reply:
Both points are valid.

Additional context: packages/ui/components/badge/UpgradeTeamsBadge.tsx is an upsell for free users, one we don't want to show for premium users which have already paid for their premium username.